src/hub.tsx

```diff
@@ -0,0 +1,197 @@
+                                You must be a Project Collection Administrator to use this tool.
+                            </div>
+                        </div>
+                    ) : (
+                        <div className="no-data-view">
+                            {this.state.loading === true
+                                ? (
+                                    <Spinner size={SpinnerSize.large} />
+                                ) : (
+                                    <ZeroData
+                                        primaryText="No banners yet..."
+                                        imageAltText="Icon"
+                                        imagePath="../static/icon.png"
+                                        actionText="Create a new banner"
+                                        actionType={ZeroDataActionType.ctaButton}
+                                        onActionClick={() => this.onAddBannerClicked()}
+                                    />
+                                )
+                            }
+                        </div>
+                    )}
+            </Page>
+        );
+    }
+
+    private getCommandBarItems(): IHeaderCommandBarItem[] {
+        return [
+            {
+                id: "add",
+                text: "Add banner",
+                isPrimary: true,
+                iconProps: {
+                    iconName: "Add",
+                },
+                onActivate: () => { this.onAddBannerClicked(); },
+            },
+            {
+                id: "delete-all",
+                text: "Delete all banners",
+                iconProps: {
+                    iconName: "Delete",
+                },
+                onActivate: () => { this.onDeleteAllClicked(); },
+            },
+            {
+                id: "info",
+                subtle: true,
+                iconProps: {
+                    iconName: "Info",
+                },
+                onActivate: () => { this.onAboutClicked(); },
+                tooltipProps: {
+                    text: "About Banner Settings",
+                },
+            },
+        ];
+    }
+
+    private async deleteAllBanners(): Promise<void> {
+        try {
+            await VstsService.instance.deleteWebGlobalMessageBanners();
+
+            // then make it reflect in the ui.
+            this.setState({ banners: [] });
+        } catch (ex) {
+            this.setState({ errorText: `There was an error deleting all banners: ${ex.message}` });
+        }
+    }
+
+    private onAddBannerClicked(): void {
+        const { banners } = this.state;
+        banners.push(new GlobalMessageBanner());
+        this.setState({ banners });
+    }
+
+    private async onAboutClicked(): Promise<void> {
+        const dialogService = await SDK.getService<IHostPageLayoutService>(CommonServiceIds.HostPageLayoutService);
+        dialogService.openMessageDialog(
+            "Copyright Microsoft 2019",

```

Comment:
I am not sure about putting year there ...

Reply:
We can use date to get the current year! Btw, In the next PR I am going to put github information in here.